### ISSUE TITLE
first 4bp trimming with NextFlex Small RNAseq library kit

### ISSUE THREAD

**Issue by hmkim** [perrykai/scripts_1_mirna_preprocess_characterization#1]
Hi, perrykai

I analyzing the NextFlex data.

Regarding to offical homepage of NextFlex (Bioo Scientific), produced reads with NextFlex kit needs a first 4bp trimming.

Do you processing that?

Thanks!
# http://www.biooscientific.com/Next-Gen-Sequencing/NEXTflex-Illumina-Small-RNA-Seq-Library-Prep-Kits

**Comment by perrykai** [comment]
Hi!

Our group chose to keep the 4bp adaptors on the 5' and 3' ends of the reads during the first quality trimming and filtering steps, as we wanted to investigate the prevalence of PCR duplicates in our small RNA seq data. After conducting that investigation we trimmed off the randomized adaptors from both ends of the reads prior to mapping to the pig reference genome.

Thanks for your question!

Kaitlyn

**Comment by hmkim** [comment]
Thanks for your reply. 

In my analysis, I don't investigate the prevalence of PCR duplicats in small rna sequencing.  Is it esstential step?

and do you mean you use the paired-end data to analyze the small rna?

When I get the pair-end sequences from sequencer (Illumina Hiseq), I use the only read 1 (forward read) to analyze the small RNA.

**Comment by perrykai** [comment]
I don't believe it would be an 'essential' step to investigate PCR duplication, and many people don't; they just trim the random adaptors and move on to mapping and quantification. We thought it would be an interesting question to ask since we actually could identify potential PCR duplicates because of these adaptors, compared to older kits like Illumina where you have no way of knowing. 

We used single-end data for this analysis, so I can't offer help with paired-end data analysis questions. I have checked forums like Biostars when I was first starting out, and they had a pretty useful discussion on single- vs paired-end seq for specifically miRNAs. I've attached a link below; be sure to read all the comments under the answer as they discuss the pros and cons of the first response. Like I said, I have had no experience with using paired-end data for this type of analysis, so I can't say any of the methods described there are necessarily correct or not, but hopefully they can provide some food for thought. 

[https://www.biostars.org/p/144292/]

Hopefully something like that can help!

Kaitlyn
